timApp/static/scripts/tim/answer/answer-browser.component.ts

```diff
@@ -282,6 +283,8 @@ export class AnswerBrowserComponent
     reviewHtml?: string;
     private answerLoader?: AnswerLoadCallback;
     pointsStep: number = 0.01;
+    autosave: boolean = true;
+    limitPoints: IPointsLimiterSettings = {min: 0, max: undefined};
```

Comment:
Tää vois vielä olla niin että min ois myös undefined, esim monivalinnoissa voi tulla negatiivisia pisteitä, sitten jos korjaa niissä vaikka jälkeenpäin manuaalisesti virheellistä pisteytystä, niin voi haluta silti antaa negatiiviset pisteet jostain tehtävästä

Reply:
Tämä muutettu nyt.